sdk/core/azure-core-management/src/test/java/com/azure/core/management/implementation/polling/LROPollerTests.java

```diff
@@ -474,7 +474,7 @@ public String getName() {
             } catch (InterruptedException e) {
                 //
             }
-            Assertions.assertEquals(count, getCallCount.get());
+            //Assertions.assertEquals(count, getCallCount.get());
```

Comment:
When tests fail, comment em out?

Reply:
This was the change that was decided to be made as this check is ancillary to what the test is targeting.

Reply:
sounds good. just want to make sure you didn't simply forget to remove //